pkgs/servers/monitoring/prometheus/xmpp-alerts.nix

```diff
@@ -2,16 +2,16 @@
 
 pythonPackages.buildPythonApplication rec {
   pname = "prometheus-xmpp-alerts";
-  version = "0.4.2";
+  version = "0.4.2+git";
 
   src = fetchFromGitHub {
     owner = "jelmer";
     repo = pname;
-    rev = version;
-    sha256 = "17aq6v4ahnga82r350kx1y8i7zgikpzmwzaacj7a339kh8hxkh63";
+    rev = "8b8d148be0bdd1b04c053d2176905bed0f053db9";
+    sha256 = "0hbhfx9k3rnhxc74idw8mnnv59w83mdq94m1x8hap8n9sc953idi";
   };
 
-  propagatedBuildInputs = with pythonPackages; [ slixmpp prometheus_client pyyaml ];
+  propagatedBuildInputs = with pythonPackages; [ slixmpp prometheus_client pytz pyyaml ];
```

Comment:
This is wrong. You only need pytz during the check phase. You are also missing the `aiohttp` runtime requirement.